src/js/views/maps/ViewfinderView.js

```diff
@@ -0,0 +1,154 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/viewfinder.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class ViewfinderView
+   * @classdesc The ViewfinderView allows a user to search for a latitude and longitude in the map view.
+   * @classcategory Views/Maps
+   * @name ViewfinderView
+   * @extends Backbone.View
+   * @screenshot views/maps/ViewfinderView.png
+   * @since 2.27.1
+   * @constructs ViewfinderView
+   */
+  var ViewfinderView = Backbone.View.extend({
+    /**
+     * The type of View this is
+     * @type {string}
+     */
+    type: "ViewfinderView",
+
+    /**
+     * The HTML classes to use for this view's element
+     * @type {string}
+     */
+    className: classNames.baseClass,
+
+    /**
+    * The events this view will listen to and the associated function to call.
+    * @type {Object}
+    */
+    events: {
+      [`change .${classNames.input}`]: 'valueChange',
+      [`click .${classNames.button}`]: 'search',
+      [`keyup .${classNames.input}`]: 'keyup',
+    },
+
+    /** 
+     * Values meant to be used by the rendered HTML template.
+     */
+    templateVars: {
+      errorMessage: "",
+      // Track the input value across re-renders.
+      inputValue: "",
+      placeholder: "Search by latitude and longitude",
+      classNames,
+    },
+
+    /**
+     * @typedef {Object} ViewfinderViewOptions
+     * @property {Map} The Map model associated with this view allowing control
+     * of panning to different locations on the map. 
+     */
+    initialize(options) {
+      this.mapModel = options.model;
+    },
+
+    render() {
+      this.el.innerHTML = _.template(Template)(this.templateVars);
+
+      this.focusInput();
+    },
+
+    focusInput() {
+      const input = this.getInput();
+      input.focus();
+      // Move cursor to end of input.
+      input.val("");
+      input.val(this.templateVars.inputValue);
+    },
+
+    getInput() {
+      return this.$el.find(`.${classNames.input}`);
+    },
+
+    getButton() {
+      return this.$el.find(`.${classNames.button}`);
+    },
+
+    /** Event handler for Backbone.View configuration. */
+    keyup(event) {
+      if (event.key === "Enter") {
+        this.search();
+      }
+    },
+
+    valueChange() {
+      this.templateVars.inputValue = this.getInput().val();
+    },
+
+    /** Event handler for Backbone.View configuration. */
+    search() {
+      this.clearError();
+
+      const coords = this.parseValue(this.templateVars.inputValue)
+      if (!coords) return;
+
+      this.model.zoomTo({ ...coords, height: 10000 /* meters */ });
+    },
+
+    /**
+     * Parse the user's input as a pair of floating point numbers. Log errors to the UI
+     * @return {{Number,Number}|undefined} Undefined represents an irrecoverable user input,
+     *   otherwise returns a latitude, longitude pair.
+     */
+    parseValue(value) {
```

Comment:
_Generally speaking_, parsing and validation of values belongs in backbone models rather than views, and we do have a GeoPoint model that we could add parsing to and use here. However, this is a pretty simple case, and so no reason to slow down getting this feature into develop! More of an FYI than a suggestion for this PR.

Reply:
I'd like to get this right. Even if I make the fix in a later commit. 
What would you expect this to look like? 
I'm guessing:
This view instantiates a GeoPoint
GeoPoint is capable of parsing a string and setting its own attributes appropriately

Does that work?

Reply:
yes, that sounds good! See the GeoPoint model [here](https://github.com/NCEAS/metacatui/blob/758ff8e3710637d9fdf406a9c2383e56ff5be877/src/js/models/maps/GeoPoint.js#L4). You could move your parsing logic there, keeping in mind that this model may eventually parse other types of inputs as well (e.g. EML). You could also use the validate method. It might be useful to change the validate method to return not only a string, but also the model attribute that's invalid. A pattern we use in other model validate methods in metacatui is to return an object in the format: 

```js
{
  latitude: "user friendly message about latitude error",
  otherAttrWithError: "another error message",
  ...
}
```

Reply:
I will do this in a follow up commit